internal/metrics/metrics.go

```diff
@@ -239,6 +242,10 @@ func NewMetrics(registry *prometheus.Registry) *Metrics {
 			},
 			[]string{"kind", "error"},
 		),
+		CircuitBreakerSettings: prometheus.NewGaugeVec(prometheus.GaugeOpts{
```

Comment:
Please also add the new metric to the `Zero()` function and re-run `make generate` to generate docs for it.

Reply:
Cool I didn't know about this